tests/unit/amo/components/TestAddon.js

```diff
@@ -1623,17 +1623,6 @@ describe(__filename, () => {
       expect(installStatus).toEqual(UNKNOWN);
     });
 
-    it('must convert all addon props to component props', () => {
-      signIn();
-      const description = 'whatever';
-      fetchAddon({ addon: { ...fakeAddon, description } });
-      const props = _mapStateToProps();
-
-      // Make sure a random addon prop gets passed as a component prop
-      // so that the withInstallHelpers HOC works.
-      expect(props.description).toEqual(description);
-    });
-
     it('must convert all installed addon props to component props', () => {
```

Comment:
This doesn't seem like a valid description for this test case anymore.

Reply:
oh, good catch!